app/src/main/java/org/simple/clinic/signature/SignatureActivity.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.signature
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.simple.clinic.R
+
+class SignatureActivity : AppCompatActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_signature)
+  }
+}
```

Comment:
Please add line break at the end of the file. You can enable this setting in Android Studio